nbs/sen2-plot.ipynb

```diff
@@ -0,0 +1,194 @@
+      "2022-08-29 17:55:37.829 | DEBUG    | metaearth.util.stac:download:50 - Downloaded data/sentinel-2-l2a/S2A_MSIL2A_20220101T184801_R070_T11SLC_20220102T161247/T11SLC_20220101T184801_B02_10m.tif\n",
+      "MB: 454it [08:20,  1.10s/it]                                                    "
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Successfully extracted 1 assets. 0 failed.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from omegaconf import OmegaConf\n",
+    "from metaearth.api import extract_assets\n",
+    "from metaearth.config import ConfigSchema\n",
+    "\n",
+    "cfg = OmegaConf.load(\"config/sen2.yaml\")\n",
+    "cfg_schema = OmegaConf.structured(ConfigSchema)\n",
+    "cfg = OmegaConf.merge(cfg_schema, cfg)\n",
+    "\n",
+    "\n",
+    "# Constrain the test to only pull one item and one band\n",
+    "for collection in cfg[\"collections\"].keys():\n",
+    "    cfg[\"collections\"][collection][\"max_items\"] = 1\n",
+    "    cfg[\"collections\"][collection][\"assets\"] = [\"B02\"]     \n",
+    "\n",
+    "successfully_extracted_assets, failed_assets = extract_assets(cfg)\n",
+    "print(\n",
+    "    f\"Successfully extracted {len(successfully_extracted_assets)} assets. {len(failed_assets)} failed.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "87c5eab1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Find the first GeoTIFF file and load it\n",
+    "# We avoid hardcoding the path here in case the API changes down the road\n",
+    "\n",
+    "fpath = Path(cfg[\"default_collection\"][\"outdir\"])\n",
+    "collection_for_gt = \"sentinel-2-l2a\"\n",
+    "gt_files = list((fpath / collection_for_gt).glob(\"**/*.tif\"))\n",
+    "\n",
+    "assert (\n",
+    "    len(gt_files) <= 1\n",
+    "), \"There were too many files pulled, `max_items` failed to constrain.\"\n",
+    "\n",
+    "\n",
+    "\n",
+    "gt_file_fpath = gt_files[0]\n",
+    "\n",
+    "data = rasterio.open(gt_file_fpath)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "37cc87aa",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
```

Comment:
It looks like the notebook fails to plot the data

Reply:
Fixed.